static/app/views/performance/trends/changeExplorer.tsx

```diff
@@ -0,0 +1,330 @@
+import {Chart} from 'sentry/views/performance/trends/chart';
+import {
+  NormalizedTrendsTransaction,
+  TrendChangeType,
+  TrendParameter,
+  TrendsStats,
+  TrendView,
+} from 'sentry/views/performance/trends/types';
+import SlideOverPanel from 'sentry/views/starfish/components/slideOverPanel';
+
+type PerformanceChangeExplorerProps = {
+  collapsed: boolean;
+  isLoading: boolean;
+  location: Location;
+  onClose: () => void;
+  organization: Organization;
+  projects: Project[];
+  statsData: TrendsStats;
+  transaction: NormalizedTrendsTransaction;
+  trendChangeType: TrendChangeType;
+  trendFunction: string;
+  trendParameter: TrendParameter;
+  trendView: TrendView;
+};
+
+type ExplorerBodyProps = {
+  isLoading: boolean;
+  location: Location;
+  organization: Organization;
+  projects: Project[];
+  statsData: TrendsStats;
+  transaction: NormalizedTrendsTransaction;
+  trendChangeType: TrendChangeType;
+  trendFunction: string;
+  trendParameter: TrendParameter;
+  trendView: TrendView;
+};
+
+type HeaderProps = {
+  transaction: NormalizedTrendsTransaction;
+  trendChangeType: TrendChangeType;
+};
+
+export function PerformanceChangeExplorer({
+  collapsed,
+  transaction,
+  onClose,
+  trendChangeType,
+  trendFunction,
+  trendView,
+  statsData,
+  isLoading,
+  organization,
+  projects,
+  trendParameter,
+  location,
+}: PerformanceChangeExplorerProps) {
+  const panelRef = useRef<HTMLDivElement>(null);
+  useOnClickOutside(panelRef, () => {
+    if (!collapsed) {
+      onClose();
+    }
+  });
+
+  const escapeKeyPressed = useKeyPress('Escape');
+
+  useEffect(() => {
+    if (escapeKeyPressed) {
+      if (!collapsed) {
+        onClose();
+      }
+    }
+  }, [escapeKeyPressed, collapsed, onClose]);
+
+  return (
+    <SlideOverPanel collapsed={collapsed} ref={panelRef}>
+      <CloseButtonWrapper>
+        <CloseButton
+          priority="link"
+          size="zero"
+          borderless
+          aria-label={t('Close Details')}
+          icon={<IconClose size="sm" />}
+          onClick={onClose}
+        />
+      </CloseButtonWrapper>
+      <PanelBodyWrapper>
+        <ExplorerBody
+          transaction={transaction}
+          trendChangeType={trendChangeType}
+          trendFunction={trendFunction}
+          trendView={trendView}
+          statsData={statsData}
+          isLoading={isLoading}
+          organization={organization}
+          projects={projects}
+          trendParameter={trendParameter}
+          location={location}
+        />
+      </PanelBodyWrapper>
+    </SlideOverPanel>
+  );
+}
+
+function ExplorerBody(props: ExplorerBodyProps) {
+  const {
+    transaction,
+    trendChangeType,
+    trendFunction,
+    trendView,
+    trendParameter,
+    isLoading,
+    location,
+    organization,
+  } = props;
+  const breakpointDate = transaction.breakpoint
+    ? new Date(transaction.breakpoint * 1000).toUTCString().replace('GMT', 'UTC')
+    : '';
```

Comment:
Should we be using timezones here? Why UTC? (also maybe this could maybe use `moment` to make this easier).

Reply:
The graph uses UTC so it was purely to be consistent. I can make it timezones if you think that's what users would find useful.